simplq/src/components/pages/Admin/admin.module.scss

```diff
@@ -8,6 +8,26 @@
   height: max-content;
 }
 
+.share {
+  .button-background {
+    background-color: $primary-color-dark;
+    border-radius: 0.6rem !important;
+  }
+
+  .popper {
+    z-index: 100;
+    background-color: #fff;
```

Comment:
This z-index as well, I didn't see any problems without it

Reply:
https://github.com/SimplQ/simplQ-frontend/blob/9c0d900cfa501fae6c4aee61f0831cbffa327f93/simplq/src/components/common/SidePanel/SidePanel.module.scss#L1-L8

As here we mentioned z-index: 10 for the side panel how when the popper opens it hides behind the side panel so I included z-index: 100 in the popper

![Screenshot from 2021-02-03 18-35-42](https://user-images.githubusercontent.com/44055337/106752225-3787de00-6650-11eb-9af4-12ab37fe8549.png)

As you can see in this image the popper hides behind side panel

Reply:
Yes removed both the z-index and everything looks fine